packages/wifi_info_flutter/wifi_info_flutter/android/src/main/AndroidManifest.xml

```diff
@@ -1,4 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
   package="io.flutter.plugins.wifi_info_flutter">
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
+    <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
```

Comment:
Please change  `<uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />` to  `<uses-permission android:name="android.permission.CHANGE_WIFI_STATE" />` .

Thank you

Reply:
I believe it's best to leave permissions to the developer. I added the permission that was in the `connectivity` plugin to avoid breaking changes.